_posts/2022-08-17-2022-summer-research-students.md

```diff
@@ -0,0 +1,46 @@
+  photo: assets/people/v.salgado.jpg
+- bio: Prateek Rout is a rising sophomore are UIUC, majoring in Mechanical Engineering. He is working on the Radio Telemetry and developing a system of 10-foot towers to track collared lizards in Arkansas and designing a housing for electronics to be attached to a drone to track boas in the Turks and Caicos Islands. He is also helping develop the molds for new fins for the SmartFin project. During the school year, he works on carbon fiber molding and designs for super-efficient cars on the Eco Illini Supermileage team at UIUC.
+  photo: assets/people/p.rout.jpg
+- bio: Melina is a rising sophomore at University of California, San Diego, majoring in Computer Science. This summer, she is working on continuing software development on the radio telemetry tracking project. This involves making changes to the current use of TCP and threading to accommodate multiple tracking towers (instead of only one in previous iterations of the project). Additionally, she is implementing the communication protocol between tracking towers and the sleep timer to integrate the sleep timer into the software system. In her free time, Melina enjoys figure skating, exploring dessert shops, and playing board games with friends.
+  photo: assets/people/m.dimitropoulou.jpg
+- bio: Samantha Prestrelski is a second-year undergraduate student at UC San Diego majoring in Mathematics-Computer Science. She is working on the Acoustic Species Identification team, using machine learning to automatically segment and classify bird vocalizations in open-source audio datasets and field recordings. In her free time, Sam likes exploring campus, baking, and spending time with friends.
+  photo: assets/people/s.prestrelski.jpg
+- bio: Robert O'Brien is a rising junior at Princeton University studying electrical and computer engineering. This summer he mainly focused on improving firmware for the Smartfin team. Additionally, he made improvements in Smartfin's hardware and fin manufacturing techniques. Robert hopes to continue working with embedded systems and conservation after he graduates. Robert likes to surf and play rugby, however back in his hometown in Virginia there aren't many waves.
+  photo: assets/people/r.obrien.jpg
+---
+This summer, we are hosting 12 students from around the US in our 2022 summer research program, finally back in person at UC San Diego.  We'll be working on measuring fish length, building SmartFins, labelling bird calls, and tracking animals.  Meet our students by reading their bios below:
+
+## Student Researchers
+{% for student in page.students %}
+<div class="floatclearfix">
+    {% if student.photo %}
+    <p><a href="{{ student.photo | absolute_url }}"><img alt="{{student.name}}" class="{{student.align}}" src="{{student.photo | resize: '180x240'}}"></a></p>{% endif %}
```

Comment:
see 2023 summer reu student page

Reply:
Fixed in 62cd4a688e0dc2b4e1ed05978c53e12fe7d08207